cmds/grpc-backend/main.go

```diff
@@ -35,8 +35,8 @@ import (
 )
 
 const (
-	// The code at this version requires a major schema version equal to 2.
-	RequiredMajorSchemaVersion = "v2"
+	// The code at this version requires a major schema version equal to 3.
+	RequiredMajorSchemaVersion = "v3"
```

Comment:
we shouldn't require this version here; it will break rid if we don't run a migration

Reply:
while scd is in development i'd only increment minor version numbres for an scd specific version, or not increment at all

Reply:
Like RID, SCD is in active use currently and we need database-binary consistency.  This PR is a breaking change for the SCD schema, so we need to make sure users who run this code against an older database are clearly alerted to the problem.  This approach mirrors what was done in b16976a to fix the similar problem introduced by 8f2687c and I agree that is not a particularly good solution which is why I [noted](https://github.com/interuss/dss/pull/334#issuecomment-648621753) that I intend to adopt the approach introduced in #328 and its successors before merging this PR.  However, we need this constraint functionality fairly soon, so we may have to use the b16976a approach in the interim while #328 and its successors are finished up.

Reply:
if we do have to merge this pr before the schema management, let's have a rid version and an SCD version

Reply:
This PR has been updated to use the recently-developed migrations (in particular, applying them to SCD in #353).